### PR TITLE
Publish recover scripts

### PR DESCRIPTION
**Pull request by srasoamiaramanana-ledger**

### CODE REVIEW

File: ledgerblue/recoverMutualAuth.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+    apdu = bytearray([0xe0, 0x04, scpv3, 0x00]) + bytearray([len(target)]) + target
+    dongle.exchange(apdu)
+
+    # Initialize authentication
+    nonce = os.urandom(8)
+    apdu = bytearray([0xe0, 0x50, 0x00, 0x00]) + bytearray([len(nonce)]) + nonce
+
+    auth_info = dongle.exchange(apdu)
+    batch_signer_serial = auth_info[0:4]
```
Comment (yhql) [inline ledgerblue/recoverMutualAuth.py:38]: pylint reports this variable is unused, is it normal?

File: ledgerblue/recoverMutualAuth.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+    # Get device certificates
+    issuer_pk = PublicKey(bytes(issuer_public_key), raw=True)
+
+    encrypted_certificate_static = dongle.exchange(bytearray.fromhex('E052000000'))
+
+    # First extract the device ephemeral public key from the device ephemeral certificate
+    certificate_ephemeral = bytearray(dongle.exchange(bytearray.fromhex('E052800000')))
+
+    certificate_header, certificate_public_key, certificate_signature_array = \
```
Comment (yhql) [inline ledgerblue/recoverMutualAuth.py:85]: pylint: 'certificate_header' unused

File: ledgerblue/recoverMutualAuth.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        extract_from_certificate(certificate_ephemeral)
+
+    # Decrypt the device static certificate
+    # Compute the shared key
+    pub_key = PublicKey(bytes(certificate_public_key), raw=True)
+    secret = pub_key.ecdh(binascii.unhexlify(ephemeral_private.serialize()), True)
+    loader = HexLoader(dongle, 0xe0, True, secret)
```
Comment (yhql) [inline ledgerblue/recoverMutualAuth.py:92]: this is not used afterwards (pylint)

File: ledgerblue/recoverMutualAuth.py [hunk 4]
```diff
@@ -0,0 +1,160 @@
+from ledgerblue.ecWrapper import PublicKey, PrivateKey
+from ledgerblue.hexLoader import HexLoader
+from ledgerblue.recoverSCP import scp_derive_key, extract_from_certificate, decrypt_certificate
+from ecpy.curves import Point, Curve
+from Crypto.Cipher import AES
+from hashlib import sha256
```
Comment (yhql) [inline ledgerblue/recoverMutualAuth.py:6]: import from ecpy, Crypto and hashlib are not used

File: ledgerblue/recoverMutualAuth.py (hunk 1 above)
Comment (srasoamiaramanana-ledger) [reply ledgerblue/recoverMutualAuth.py]: Yes, batch_signer_serial is used to select the right public key to verify the device's certificate but it isn't relevant here, so I just removed it.

File: ledgerblue/recoverBackup.py [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+import os
+
+
+def get_argparser():
+    parser = argparse.ArgumentParser(description="Backup 3 shares of a seed.")
+    parser.add_argument("--targetId", help="The device's target ID (default is Nano X)", type=auto_int)
+    parser.add_argument("--rootPrivateKey", help="The private key of the Certificate Authority")
+    parser.add_argument("--issuerPublicKey", help="The public key of the Issuer (used to verify the certificate of "
+                                                  "the device)")
+    parser.add_argument("--nWords", help="The number of words in the mnemonic (default is 24 words)", type=auto_int)
```
Comment (YBadiss) [inline ledgerblue/recoverBackup.py:20]: Any chance we can change this to `numberOfWords`? `nWords` has a bad ring to it 😅

**Review by yhql — APPROVED** [review]

**Pull request merged**